README.md

```diff
@@ -91,30 +91,91 @@ Browser Mod has a number of services you can call to cause things to happen in t
 
 ## Popup card
 
-A popup card can be used to replace the more-info dialog of an entity with something of your choosing.
+Popup cards can be used to replace the more-info dialog of an entity, or to be used as a template for [`browser_mod.popup` service](documentation/services.md), or both!
 
-To use it, add a "Custom: Popup card" to a dashboard view via the GUI, pick the entity you want to override, configure the card and set up the popup like for the [`browser_mod.popup` service](documentation/services.md).
+### Popup card - replace more-info dialog of an entity
+
+A popup card can be used to replace the more-info dialog of an entity with something of your choosing. The entity can be targeted by entity, area, label or device.
```

Comment:
nit: 'the entity can be targeted' -> 'the popup can be targeted'

Or perhaps make the sentence more active as in...

You can use a popup card to replace the built-in more-info dialog.  When you open a more-info dialog for an entity that matches the popup card's target filter, the popup card will be shown instead of the built-in more-info dialog.  The popup card's target filter can be an entity id, area, label, or device.

Reply:
Taking your suggestion I will rewrite as

>You can use a popup card to replace the built-in more-info dialog. When you open a more-info dialog for an entity that matches the popup card's target filter, the popup card will be shown instead of the built-in more-info dialog. The popup card's target filter can be one or more of entity id, area, label, or device.